### PR TITLE
Workbox homepage

### PR DESCRIPTION
**Pull request by gauntface**
New homepage for Workbox.

### CODE REVIEW

File: src/content/en/tools/workbox/next/_index.yaml [hunk 1]
```diff
@@ -6,26 +6,210 @@ landing_page:
...
+        }
+      </style>
+      <div class="workbox-columns">
+        <div class="workbox-columns__column">
+          <div class="row-icon">
+            <img src="../images/performance.svg" alt="Performance Icon" />
+          </div>
+          <div class="row-details">
+            <h3>Performance</h3>
+            <p>Stop waiting on the network! You can improve your web apps performance by caching and serving your files, powered by a service worker.</p>
```
Comment (philipwalton) [inline src/content/en/tools/workbox/next/_index.yaml]: s/apps/app's/

File: src/content/en/tools/workbox/next/_index.yaml [hunk 2]
```diff
@@ -6,26 +6,210 @@ landing_page:
...
+        <p>Workbox is a library that bakes in a set of best practices and removes the boilerplate every  developer writes when working with <a href="https://developers.google.com/web/fundamentals/primers/service-workers/">service workers</a>.</p>
+
+        <ul>
+        <li>Precaching</li>
+        <li>Runtime caching</li>
+        <li>Strategies</li>
+        <li>Request routing</li>
+        <li>Background sync</li>
+        <li>Helpful debugging</li>
+        <li>Greate flexibility and feature set than sw-precache and sw-toolbox</li>
```
Comment (philipwalton) [inline src/content/en/tools/workbox/next/_index.yaml]: s/Greate/Greater/

File: src/content/en/tools/workbox/next/_index.yaml [hunk 3]
```diff
@@ -6,26 +6,210 @@ landing_page:
...
+        </ul>
+
+  - custom_html: >
+      <h2>Whats the API Like?</h2>
```
Comment (philipwalton) [inline src/content/en/tools/workbox/next/_index.yaml]: /Whats/What's/

File: src/content/en/tools/workbox/next/_index.yaml [hunk 4]
```diff
@@ -6,26 +6,210 @@ landing_page:
...
+      <p>Below is a few examples of the Workbox API demonstrating some of
```
Comment (philipwalton) [inline src/content/en/tools/workbox/next/_index.yaml]: s/is/are/

File: src/content/en/tools/workbox/next/_index.yaml [hunk 5]
```diff
@@ -6,26 +6,210 @@ landing_page:
...
+      workbox.routing.registerRoute(
+        /\.(?:js|css)$/,
+        workbox.strategies.staleWhileRevalidate(),
+      );
+      </pre>
+
+      <h3>Cache Images</h3>
+      <p>Images carry most of the weight for a web page. Use this rule to
+      serve them quickly from the cache, while making sure you don’t cache
+      them indefinitely, consuming your user’s stoage.</p>
```
Comment (philipwalton) [inline src/content/en/tools/workbox/next/_index.yaml]: s/user’s stoage/users’ storage/

File: gulp-tasks/test.js [hunk 6]
```diff
@@ -368,7 +368,7 @@ function getFiles() {
     if (IS_TRAVIS) {
       cmd += '$(git merge-base FETCH_HEAD master) FETCH_HEAD';
     } else {
-      cmd += '$(git merge-base master HEAD)';
+      cmd += '$(git merge-base origin/master HEAD)';
```
Comment (gauntface) [inline gulp-tasks/test.js]: @petele I had to change to this so the comparison of local files would test against remote master not the current local master.

Comment (petele) [reply gulp-tasks/test.js]: Did you test for local changes as well as for changes on Travis?

**Pull request merged**